src/templates/kit/screens/VotingScreen.js

```diff
@@ -8,6 +8,7 @@ import {
   Navigation,
```

Comment:
Rather than make the changes here, let's move the changes to a new component under `open-enterprise`.

The changes are starting to assume data coming from a previous screen, and so are less generic.